wagtail/admin/templates/wagtailadmin/login.html

```diff
@@ -82,7 +82,7 @@ <h1>{% block branding_login %}{% trans "Sign in to Wagtail" %}{% endblock %}</h1
             </ul>
 
             {% if show_password_reset %}
-                <p class="help"><a href="{% url 'wagtailadmin_password_reset' %}">{% trans "Forgotten it?" %}</a></p>
+                <p class="help" aria-label="{% trans 'Reset your password' %}"><a href="{% url 'wagtailadmin_password_reset' %}">{% trans "Forgotten it?" %}</a></p>
```

Comment:
The aria-label should be on the a tag, as that is what will be focused by the screen reader.